packages/trip-form/src/MetroModeSelector/index.tsx

```diff
@@ -0,0 +1,407 @@
+import {
+  arrow,
+  FloatingFocusManager,
+  offset,
+  safePolygon,
+  shift,
+  useClick,
+  useDismiss,
+  useFloating,
+  useHover,
+  useInteractions,
+  useRole
+} from "@floating-ui/react";
+import { ModeButtonDefinition } from "@opentripplanner/types";
+import { CaretDown } from "@styled-icons/fa-solid/CaretDown";
+import { CaretUp } from "@styled-icons/fa-solid/CaretUp";
+import React, { ReactElement, useCallback, useRef, useState } from "react";
+import { FormattedMessage } from "react-intl";
+import styled, { css } from "styled-components";
+
+import SubSettingsPane, { defaultMessages } from "./SubSettingsPane";
+
+const invisibleCss = css`
+  clip: rect(0, 0, 0, 0);
+  height: 0;
+  overflow: hidden;
+  position: absolute;
+  width: 0;
+`;
+
+const InvisibleA11yLabel = styled.span`
+  display: inline-block;
+  ${invisibleCss}
+`;
+
+const ModeBar = styled.fieldset`
+  border: none;
+  display: inline-flex;
+  gap: 0 3px;
+  margin: 0 4px 0 0;
+  padding: 0;
+
+  /* <legend> is not shown visually. */
+  & > legend {
+    ${invisibleCss}
+  }
+`;
+
+const accentColor = "#084c8d";
```

Comment:
Can we make this configurable/importable from props?

Reply:
See https://github.com/ibi-group/configurations/pull/379/files#r1153514224 for an example on how to do this.

Reply:
Added some props for override